src/main/scala/org/ergoplatform/nodeView/history/storage/HistoryStorage.scala

```diff
@@ -19,76 +19,68 @@ import scala.util.{Failure, Success, Try}
   * @param objectsStore - key-value store, where key is id of ErgoPersistentModifier and value is it's bytes
   * @param config       - cache configs
   */
-class HistoryStorage(indexStore: Store, objectsStore: ObjectsStore, config: CacheSettings) extends ScorexLogging
-  with AutoCloseable with ScorexEncoding {
+class HistoryStorage(indexStore: LDBKVStore, objectsStore: LDBKVStore, config: CacheSettings)
+  extends ScorexLogging
+    with AutoCloseable
+    with ScorexEncoding {
 
   private val modifiersCache = CacheBuilder.newBuilder()
     .maximumSize(config.modifiersCacheSize)
     .build[String, ErgoPersistentModifier]
 
   private val indexCache = CacheBuilder.newBuilder()
     .maximumSize(config.indexesCacheSize)
-    .build[ByteArrayWrapper, ByteArrayWrapper]
+    .build[ByteArrayWrapper, Array[Byte]]
 
-  def modifierById(id: ModifierId): Option[ErgoPersistentModifier] = {
-    Option(modifiersCache.getIfPresent(id)) match {
-      case Some(e) =>
-        log.trace(s"Got modifier $id from cache")
-        Some(e)
-      case None =>
-        objectsStore.get(id).flatMap { bBytes =>
-          HistoryModifierSerializer.parseBytesTry(bBytes) match {
-            case Success(pm) =>
-              log.trace(s"Cache miss for existing modifier $id")
-              modifiersCache.put(id, pm)
-              Some(pm)
-            case Failure(_) =>
-              log.warn(s"Failed to parse modifier ${encoder.encode(id)} from db (bytes are: ${Algos.encode(bBytes)})")
-              None
-          }
+  def modifierById(id: ModifierId): Option[ErgoPersistentModifier] =
```

Comment:
log.trace(s"Got modifier $id from cache") has been lost during "refactoring"

Reply:
It was not lost but removed as modifiers cache is already tested so there is no need in such logging.